sql-parser/src/main/java/io/crate/sql/parser/AstBuilder.java

```diff
@@ -1723,6 +1732,20 @@ private static String getUnquotedText(@Nullable ParserRuleContext context) {
         }
     }
 
+    private String getTrimMode(Token type) {
+        if (type == null) {
+            return "BOTH";
+        }
+        switch (type.getType()) {
+            case SqlBaseLexer.BOTH:
```

Comment:
Would it make sense to have the new enum in the sql-parser module and use it here already?